src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PurposesApiMarshallerImpl.scala

```diff
@@ -1,14 +1,26 @@
 package it.pagopa.interop.backendforfrontend.api.impl
 
 import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
-import akka.http.scaladsl.marshalling.ToEntityMarshaller
+import akka.http.scaladsl.marshalling.{Marshaller, ToEntityMarshaller}
+import akka.http.scaladsl.model.ContentTypes
 import it.pagopa.interop.backendforfrontend.api.PurposesApiMarshaller
 import it.pagopa.interop.backendforfrontend.model._
 import spray.json.DefaultJsonProtocol
+import java.io.File
+import java.nio.charset.StandardCharsets
+import scala.io.{BufferedSource, Codec}
 
 object PurposesApiMarshallerImpl extends PurposesApiMarshaller with SprayJsonSupport with DefaultJsonProtocol {
 
   override implicit def toEntityMarshallerProblem: ToEntityMarshaller[Problem] = entityMarshallerProblem
 
   override implicit def toEntityMarshallerPurposes: ToEntityMarshaller[Purposes] = sprayJsonMarshaller[Purposes]
+
+  override implicit def toEntityMarshallerFile: ToEntityMarshaller[File] =
+    Marshaller.withFixedContentType(ContentTypes.`application/octet-stream`) { f =>
```

Comment:
we could centralize this marshaller like the `entityMarshallerProblem`, as it is/will be used in several services